### PR TITLE
Internal: Remove common from atomic widgets [EDS-221]

### PR DESCRIPTION
**Pull request by Omerisra6**

### CODE REVIEW

File: assets/dev/js/editor/editor-base.js [hunk 1]
```diff
@@ -284,7 +284,7 @@ export default class EditorBase extends Marionette.Application {
 				return false;
 			}
 
-			if ( ! this.widgetsCache[ widgetType ].commonMerged ) {
+			if ( ! this.widgetsCache[ widgetType ].commonMerged && ! this.widgetsCache[ widgetType ].atomic_controls ) {
```
Comment (StyleShit) [inline assets/dev/js/editor/editor-base.js:287]: maybe extract `this.widgetsCache[ widgetType ]` to a variable?

File: tests/playwright/sanity/atomic-widgets.test.ts [hunk 2]
```diff
@@ -42,11 +47,5 @@ test.describe( 'Atomic Widgets', () => {
 			} );
 		} );
 	} );
-
-	async function enableRequiredExperiments() {
-		await wpAdmin.setExperiments( {
-			atomic_widgets: 'active',
-		} );
-	}
```
Comment (StyleShit) [inline tests/playwright/sanity/atomic-widgets.test.ts:50]: don't you need to reset the experiments? 
it's the same server, so you might introduce side effects

File: includes/widgets/atomic-heading.php [hunk 3]
```diff
@@ -12,19 +12,22 @@ public function get_icon() {
...
-		$content = $this->get_settings()['title'] ?? 'Hello, World!';
+		$tag = $this->get_settings( 'tag' ) ?? 'h2';
+		$title = $this->get_settings( 'title' ) ?? 'Hello, World!';
+
+		$escaped_tag = Utils::validate_html_tag( $tag );
+		$escaped_title = esc_html( $title );
 
 		// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
-		echo "<$tag>$content</$tag>";
+		echo "<$escaped_tag>$escaped_title</$escaped_tag>";
```
Comment (StyleShit) [inline includes/widgets/atomic-heading.php:30]: do you still need the comment?

Comment (StyleShit) [inline includes/widgets/atomic-heading.php:30]:
```suggestion
		echo "<{$escaped_tag}>{$escaped_title}</{$escaped_tag}>";
```

**Review by StyleShit — CHANGES_REQUESTED** [review]

File: includes/widgets/atomic-heading.php (hunk 3 above)
Comment (Omerisra6) [reply includes/widgets/atomic-heading.php:30]: Yes...

File: assets/dev/js/editor/editor-base.js (hunk 1 above)
Comment (Omerisra6) [reply assets/dev/js/editor/editor-base.js:287]: I don't think it is necessary.

File: tests/playwright/sanity/atomic-widgets.test.ts (hunk 2 above)
Comment (Omerisra6) [reply tests/playwright/sanity/atomic-widgets.test.ts:50]: I reset the experiments on line 29 (afterAll)

File: includes/widgets/atomic-heading.php (hunk 3 above)
Comment (Omerisra6) [reply includes/widgets/atomic-heading.php:30]: why?

**Review by StyleShit — APPROVED** [review]

**Pull request merged**